client-side.md

```diff
@@ -4,6 +4,15 @@ These are recommendations from the experiences of multiple teams
 across the Guardian. Feel free to not follow of them, but also feel
 free to justify why not.
 
+## `@guardian` packages
+### Publishing
+- Publish packages in ES2020 JavaScript
```

Comment:
Since we use TypeScript so frequently, do you think it would be worth mentioning or linking to _how_ to do this in that context? i.e. the `target` param in [TypeScript config options](https://www.typescriptlang.org/docs/handbook/compiler-options.html).

Reply:
yeah great idea, will add that